Assets/Scripts/EOSPlayerDataStorageManager.cs

```diff
@@ -29,6 +29,8 @@
 
 using Epic.OnlineServices;
 using Epic.OnlineServices.PlayerDataStorage;
+using static Codice.Client.Common.WebApi.WebApiEndpoints;
```

Comment:
Is this necessary for the fixes?

Reply:
Nope, just removed them. Must have been introduced between my experimentation and the finished fix.